script/github-actions/codecoverage-format-report.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const path = require('path');
+
+const codecoverageReport = fs.readFileSync(
+  path.join(__dirname, '../../test-results/coverage_report.txt'),
+  'utf8',
+);
+
+// eliminiate unnecessary lines & spaces from coverage report
+const regex = /([^A-Za-z0-9\/()%-])+\s/g; // eslint-disable-line no-useless-escape
```

Comment:
I don't think you need to escape `/` in the group.

```suggestion
const regex = /([^A-Za-z0-9/()%-])+\s/g; // eslint-disable-line no-useless-escape
```

If you actually mean to match the backslash, perhaps that part could be `/\` instead of `\/` to avoid disabling the ESLint rule.